src/components/Drawers/DrawerItem.js

```diff
@@ -12,7 +12,7 @@ const styles = () => ({
 
 function DrawerItem(props) {
   const { classes } = props;
-  const active = props.to.match(props.location.pathname.split("/")[1]);
+  const active = props.to === `/${props.location.pathname.split("/")[1]}`;
```

Comment:
This is to fix the double active tabs, right?

Reply:
yes